packages/plugin-ext/src/main/browser/plugin-frontend-contribution.ts

```diff
@@ -54,15 +68,174 @@ export class PluginApiFrontendContribution implements CommandContribution {
             execute: () => this.pluginExtDeployCommandService.deploy()
         });
 
-        // this command only for compatibility reason
-        commands.registerCommand({ id: 'workbench.action.closeActiveEditor' }, {
-            execute: () => commands.executeCommand('core.close.tab')
-        });
-
         commands.registerCommand(OpenUriCommandHandler.COMMAND_METADATA, {
             execute: (arg: URI) => this.openUriCommandHandler.execute(arg),
             isVisible: () => false
         });
+
```

Comment:
I thought these commands were going to 
https://github.com/theia-ide/theia/blob/68795ecf308155c85cf6107239201e784976c388/packages/plugin-ext/src/plugin/command-registry.ts#L87

https://github.com/theia-ide/theia/blob/68795ecf308155c85cf6107239201e784976c388/packages/plugin-ext/src/plugin/type-converters.ts#L437

Reply:
Could you elaborate? I don't know this code. `editor.action.showReferences` is from monaco. Command which I've added they are not contributed by monaco. Also they cannot be implemented in the plugin host process, they need access to the shell and editor manager.

Reply:
https://github.com/theia-ide/theia/pull/4193#issuecomment-459224709

Reply:
it is this comment.

It seems there are different places where there is mapping/aliases between vscode commands and theia commands so might be good to have a single location

Reply:
I can move it here then: https://github.com/theia-ide/theia/blob/30eaeb02cca616f11d59988ba562eccc3891a6ae/packages/plugin-ext/src/main/browser/command-registry-main.ts#L70

Reply:
I'm fine there or in a command-contribution separate class where we only register these commands like for vscode https://github.com/theia-ide/theia/blob/master/packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts as it's kind easy to know what to find in this class.

Reply:
ok plugin-vscode-commands-contribution sounds good as well